test/TestUtils/TestProjectFixture.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Microsoft.DotNet.Cli.Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test
+{
+    public class TestProjectFixture
```

Comment:
I'm fine keeping this logic separate.  I just needed to understand more what the intention of the class was and how it differed from TestProject.

---
In reply to: [66350076](https://github.com/dotnet/core-setup/pull/97#discussion_r66350076) [](ancestors = 66350076,66348239,66107984,66102620)